app/assets/javascripts/osom-tables.js

```diff
@@ -117,6 +132,32 @@
     }
   }
 
+  function save_checkboxes(container) {
```

Comment:
It's unclear to me what it means to "save checkboxes". Might be worth calling this, e.g. `store_checked_ids_in_data_attribute` or something?